packages/dashboard/src/app/views/shared/savedTemplateGridView.js

```diff
@@ -129,7 +121,7 @@ const SavedTemplateGridView = ({
                 itemRefs.current[template.id] = el;
               }}
             >
-              <FocusableGridItem
```

Comment:
this never gets seen. You can't save stories so you only get the empty view even if you enable this sad old experiment. I wanted to delete some of these older components so replacing the instances here with standard elements.